doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -615,7 +617,9 @@ Edge Impulse integration
 Memfault integration
 --------------------
 
-|no_changes_yet_note|
+* Added a new feature to automatically post coredumps to Memfault when network connectivity is available.
```

Comment:
The entry could be moved to the library section - https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/releases_and_maturity/releases/release-notes-changelog.html#debug_libraries as the feature is mentioned there.

Reply:
Not sure, this kconfig is a part of NCSs integration with Memfault. It only adds a library to the existing suite of c files that we use to integrate Memfault.